index.js

```diff
@@ -3,11 +3,7 @@ const bodyParser = require('body-parser')
 const express = require('express')
 const fs = require('fs')
 const https = require('https')
-const session = require('express-session')
```

Comment:
Pretty much everything deleted from this file moved into either `routes.js` or `dashboard.js`.